build.ps1

```diff
@@ -60,6 +60,18 @@ function NugetPack {
     }
 }
 
+function PackSelfContainProject {
```

Comment:
will refactor later...

---
In reply to: [95934194](https://github.com/dotnet/docfx/pull/1167#discussion_r95934194) [](ancestors = 95934194,95931939,95931418)